python/dgl/distributed/dist_context.py

```diff
@@ -29,6 +31,8 @@ def _init_rpc(ip_config, max_queue_size, net_type, role):
     ''' This init function is called in the worker processes.
     '''
     try:
+        os.environ['OMP_NUM_THREADS'] = '1'
```

Comment:
Changing `OMP_NUM_THREADS` in a running process doesn't change the number of OMP threads.
https://software.intel.com/content/www/us/en/develop/documentation/mkl-linux-developer-guide/top/managing-performance-and-memory/improving-performance-with-threading/changing-the-number-of-openmp-threads-at-run-time.html

Reply:
so we should create an CAPI to change the number of OMP threads?

Reply:
I guess so.